java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/SinkWriterStreamObserver.java

```diff
@@ -41,18 +41,48 @@ public class SinkWriterStreamObserver
     private boolean finished = false;
 
     private boolean epochStarted;
-    private long currentEpoch;
+    private Long currentEpoch;
```

Comment:
not familiar with Java, whats the difference

Reply:
long is a primitive type, while Long is an object. Using Long we can use null to represent the uninitialized state.